src/ui/components/ExplorerHome/ExplorerHome.js

```diff
@@ -354,22 +349,51 @@ export const ExplorerHome = ({
     </div>
   );
 
-  const formatInterval = (interval) =>
-    formatTimestamp(interval.startTimeMs, {
-      month: "short",
-      day: "numeric",
-      hour: undefined,
-      minute: undefined,
-      year: "numeric",
-    }) +
-    " to " +
-    formatTimestamp(interval.endTimeMs, {
-      month: "short",
-      day: "numeric",
-      hour: undefined,
-      minute: undefined,
-      year: "numeric",
-    });
+  const formatInterval = (interval) => {
+    const formatWithYear = (timestamp) =>
+      formatTimestamp(timestamp, {
+        month: "short",
+        day: "numeric",
+        hour: undefined,
+        minute: undefined,
+        year: "numeric",
+      });
+
+    const formatWithoutYear = (timestamp) =>
+      formatTimestamp(timestamp, {
+        month: "short",
+        day: "numeric",
+        hour: undefined,
+        minute: undefined,
+        year: undefined,
+      });
+
+    const isTodayOrLater = (someDate) => {
+      const today = new Date();
+      today.setHours(0);
+      today.setMinutes(0);
+      today.setSeconds(0);
+      today.setMilliseconds(0);
+
+      return someDate >= today;
+    };
+
+    const currentYear = new Date().getFullYear();
+    const startDate = new Date(interval.startTimeMs);
+    const endDate = new Date(interval.endTimeMs);
+
+    const startTime =
+      startDate.getFullYear() === currentYear
```

Comment:
Isnt this supposed to compare the startDate year to the _endDate year_, not the current year.

Reply:
Ah good point. In my head I was assuming end date year would always be the current year, but I can see how that would not necessarily be the case. Good catch, and thank you.